docs/csharp/language-reference/operators/member-access-operators.md

```diff
@@ -206,6 +209,23 @@ You can omit any of the operands of the `..` operator to obtain an open-ended ra
 
 :::code language="csharp" source="snippets/shared/MemberAccessOperators.cs" id="RangesOptional":::
 
+The following table shows various ways to express collection ranges:
+
+| Range operator expression | Description                                                                          |
+|---------------------------|--------------------------------------------------------------------------------------|
+| `..`                      | All values in the collection.                                                        |
+| `..{end}`                 | Values from the start to the `{end}` exclusively.                                    |
+| `{start}..`               | Values from the `{start}` inclusively to the end.                                    |
+| `{start}..{end}`          | Values from the `{start}` inclusively to the `{end}` exclusively.                    |
+| `^{start}..`              | Values from the `{start}` inclusively to the end counting from the end.              |
+| `..^{end}`                | Values from the start to the `{end}` exclusively counting from the end.              |
+| `{start}..^{end}`         | Values from `{start}` inclusively to `{end}` exclusively counting from the end.      |
+| `^{start}..^{end}`        | Values from `{start}` inclusively to `{end}` exclusively both counting from the end. |
```

Comment:
My another concern here is using the words "start" and "end" in two different meanings: (1) start and end of a collection, (2) names of range boundaries. If somebody read out this table to me, I doubt I would understand it correctly. I must SEE the text to understand it correctly. So it's not audio-friendly. 

What do you think about `a..b` or `s..e` notation (then also no need in braces, I guess)?

Reply:
I agree. At first, I thought it would be nice to use the same wording as the boundary terms themselves, that way you could quickly identify which are variables in the expression. I wanted something more descriptive than just a single letter, at first I had `{n}`, or `{n1}` and `{n2}`.

Reply:
I agree that more descriptive names would be nice, but `start` and `end` may be confusing.
By the way, why do you use the braces? One may think that those are part of an expression as well (or I'm overcautious here?)

Reply:
I removed them, but they're pretty common when calling out parameters in expressions, no?

Reply:
@IEvangelist thank you; hm, I don't how frequent they are